src/webfaf2/problems.py

```diff
@@ -79,7 +82,6 @@ def query_problems(db, hist_table, hist_column,
     if associate_id:
         assoc_query = (
             db.session.query(ProblemComponent.problem_id.label('problem_id'))
-            .distinct(ProblemComponent.problem_id)
```

Comment:
This for some reason reverts my earlier commit https://github.com/abrt/faf/commit/8a28a3a07c06daa7a11bcacff3a90f9c7d279100 .